test-fixtures/src/bin/grpc.rs

```diff
@@ -1,13 +1,13 @@
-use fastly::{Backend, Error, Request};
```

Comment:
Huh, the formatting step in CI must be missing these if rustfmt changed these despite previous successful commits. Something to worry about later